iotlabcli/integration/test_cli_tools_help.py

```diff
@@ -0,0 +1,36 @@
+import pytest
```

Comment:
Don't take it bad but I think that is already covered by https://github.com/iot-lab/cli-tools/blob/master/tox.ini#L28 in the CI.
It really seems to me that you are duplicating what already exists.
The output of the `--help` option can (should) be tested in the parser test module.